testpar/t_subfiling_vfd.c

```diff
@@ -2892,6 +2892,11 @@ main(int argc, char **argv)
     bool     must_unset_config_dir_env       = false;
     int      required                        = MPI_THREAD_MULTIPLE;
     int      provided                        = 0;
+    char    *subfiling_subfile_prefix_saved;
```

Comment:
Same fix here?  (Either not saving the values or strdup'ing them)

Reply:
Do you think saving the values is necessary? I wasn't sure. If you don't think so, I can just remove it there too.

Reply:
For the time being, we don't really need to save the values since this is the last set of tests that run. If another set of tests is interested in the values from the environment in the future we may need to save the values, but otherwise it's fine to remove them for now.

Reply:
Okay, I can remove it from this test too.